config/taxonomy/Makefile

```diff
@@ -0,0 +1,12 @@
+GLANG ?= "go"
+all: clean compile gen
+compile:
+	go run ../../main.go taxonomy compile --base ./base/base.yaml --out ./objects/taxonomy.json \
+	./layers/module/actions.yaml \
+	./layers/catalog/locations.yaml \
+	./layers/catalog/tags.yaml \
+		--codegen  
```

Comment:
For this PR no layers, just base taxonomy.

Reply:
Done